zerver/tests/test_import_export.py

```diff
@@ -594,48 +594,60 @@ def test_zulip_realm(self) -> None:
     def test_export_realm_with_exportable_user_ids(self) -> None:
         realm = Realm.objects.get(string_id="zulip")
 
-        cordelia = self.example_user("iago")
```

Comment:
This test was kind of broken, due to the `exportable_user_ids` codepath being unused code in reality. We have plans to unite `exportable_user_ids` with "consented users", so this test gets cleaned up with that in mind; additionally testing that the bug targeted by this commit is fixed.

In particular, this test now does an export with member consent; since `exportable_user_ids` doesn't conceptually even make sense anymore outside of that context.